ops/model.py

```diff
@@ -107,6 +110,8 @@
     })
 
 
+StrOrPath = typing.Union[str, Path]
```

Comment:
The type hint for the `file` arg of `open` is `str | bytes | PathLike[str] | PathLike[bytes]`.
Not sure if matching it is needed here.

Reply:
I think we can expand the types later if there seems to be demand.  This was mostly a quick QoL tweak to allow pathlib paths while still supporting (very convenient) strings as are used in several other methods.